tests/e2e/env/webpack-alias.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+module.exports = {
+	'@woocommerce/e2e-tests': path.resolve(
```

Comment:
Sorry if this is totally wrong! Wouldn't this resolve to `tests/e2e/env/node_modules/woocommerce/tests/e2e` (at least, in our repo)

Reply:
That would be true if there wasn't a `docker-compose.yml` in the root of the project.

Reply:
No worries then, I thought `__dirname` just always returned the directory of the JS file in question!

Reply:
The `path.resolve()` climbs the directory tree looking for the config file in any of those folders.